chrome-extension/vite.config.mts

```diff
@@ -36,6 +36,7 @@ export default defineConfig({
       fileName: 'background',
     },
     outDir,
+    emptyOutDir: true,
```

Comment:
The outdir in this config is the `dist` folder, so how about use emptyOutDir to `false` to avoid issues with race conditions?

Actually, I think the problem occurs when I run the build or dev command locally.

```suggestion
    emptyOutDir: false,
```

Reply:
Yeah, that's need to be false. because of the dist in root, my bad.

Thanks you for the second look :)